jupyter-book/multi_omics/1_0_scATAC_intro.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<p>I'd be happy if we had 2-3 detailed sentences on why gene activity + epigenetics matters with the video being more of a supplement. Currently (to me at least) it feels like I HAVE TO watch the video to understand the reasoning.</p><p> </p><blockquote>After walking you through the preprocessing steps this analysis will allow us to: 1) characterize cell identity with an orthogonal approach to scRNA-seq analysis 2) identify cell state specific transcriptional regulators 3) link gene expression to sequence features 4) disentagle epigenetic mechanisms driving cell differentiation and disease states </blockquote><p> </p><p>Please make this a markdown listing</p><p> </p><p>-&gt;</p><p> </p><p>After walking you through the preprocessing steps this analysis will allow us to:</p><p> </p><p>1. Characterize cell identity with an orthogonal approach to scRNA-seq analysis</p><p>2. Identify cell state specific transcriptional regulators </p><p>3. Link gene expression to sequence features </p><p>4. Disentagle epigenetic mechanisms driving cell differentiation and disease states </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>